evap/staff/tests/test_views.py

```diff
@@ -35,6 +35,23 @@ def test_num_queries_is_constant(self):
             self.app.get(self.url, user="staff")
 
 
+class TestSemesterCourseOperationView(ViewTest):
+    @classmethod
+    def setUpTestData(cls):
+        mommy.make(UserProfile, username='staff', groups=[Group.objects.get(name='Staff')])
+
+    def test_operation_start_evaluation(self):
+        sem = mommy.make(Semester,pk=1)
+        mommy.make(Course, pk=1, state='approved', semester=sem)
+
+        url = '/staff/semester/1/courseoperation?course=1&operation=startEvaluation'
```

Comment:
this view does indeed need some setup for the simple 200_OK check. So I decided not to set test_users